webapp/templates/process_invoices_confirm_final_run.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'base.html' %}
+
+{% block title %}Confirm final run{% endblock title %}
+
+{% block header %}
+{% endblock header %}
+
+{% block content %}
+<h1>Confirm final run</h1>
+<h3>Are you sure you want to execute a <em>final</em> run?</h2>
+
+<p>
+    <a class="btn button-primary" href="{{ url_for('process_invoices') }}">No</a>
+    <a class="btn button-primary warn" href="{{ url_for('process_invoices_run_execute', run_type='final') }}">
+        Execute a <strong>final</strong> run
+    </a>
```

Comment:
I think this button is _right_ on the line of maybe too busy / too many emojis + colors.  But, I like it!  Perhaps stakeholders will have feedback about language, but it certainly adds a check to the process.

Reply:
@jonavellecuerdo - good point red color blindness!  I had not considered that.  +100 to style guide usage + thoughtful implementation.  I hereby retract all "busy" concerns.

Reply:
The style guide has been helpful throughout this process! :) The "busy" feeling could be that the phrase on the button is a bit long, but I think a little wordiness is fine considering we're _really_ trying to prevent "final" runs. 😅